service/ExchangeService.js

```diff
@@ -10,19 +10,22 @@ var log = new Logger(AppConfig.logLevel);
 
 var fetchExchangeData = function(exchange, url, callback){
 	request.get(url, function (err, response, body) {
-		if ( !err && response.statusCode == 200 ){
+				if ( !err && response.statusCode == 200 ){
```

Comment:
Maybe correct spacing here as it looks off (are we using a linter or something which has ability to format or check formatting)?

Reply:
I think we should start using eslint ...